src/componentes/Header.js

```diff
@@ -4,13 +4,13 @@ import "../css/Header.css";
 import Logo from "../imgs/alura.svg";
 
 export default class Header extends Component {
-  render() {
-    return (
-      <header>
-        <Link to="/">
-          <img src={Logo} alt="Logo" />
-        </Link>
-      </header>
-    );
-  }
+    render() {
+        return (
+        <header>
+            <Link to="/">
```

Comment:
Acho que aqui dá pra adicionar um outro link para fazer um novo lance, né?

Reply:
Da sim, até porque o header está apenas com o logo da alura. Poderia deixar a logo da alura no canto esquerdo e colocar o botão de lance na direita do header. Você acha que ficaria bom?

Reply:
Acho sim! Por semântica, se for colocar o menu aí só não esquece de colocar dentro de um nav 😊